.github/workflows/test.yml

```diff
@@ -22,5 +24,18 @@ jobs:
           pnpm clean
           pnpm build
           pnpm test-packaging
+  release:
+    runs-on: ubuntu-latest
+    needs: test
+    if: ${{ contains('refs/tags', github.ref) == 'true' }}
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+
+      - name: Install
+        uses: ./.github/workflows/install
+
+      - name: Release
+        run: pnpm publish --access public --no-git-checks
         env:
-          CI: true
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN_ELEVATED }}
```

Comment:
https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry